release/generate_manifest.sh

```diff
@@ -318,6 +292,8 @@ pushd "${CLONE_DIR}"
   #copy the needed files
   gsutil cp "${MANIFEST_FILE}" "${BASE_MANIFEST_URL}"
   gsutil cp "${MANIFEST_FILE}" "${MANIFEST_URL}"
+  gsutil cp "${MANIFEST_FILE}" "${GCS_BUILD_PATH}"
```

Comment:
Why is the manifest file needed in different places?

And why
  BASE_MANIFEST_URL="gs://istio-release-pipeline-data/release-tools/${BRANCH}-manifest.xml"
  BASE_MASTER_MANIFEST_URL="gs://istio-release-pipeline-data/release-tools/master-manifest.xml"